src/style/app.module.css

```
@@ -1546,6 +1687,8 @@ hr {
 .modalHeader {
   border: none;
   padding-bottom: 0;
+  background-color: var(--bs-primary) !important;
+  padding: 1rem 1rem;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-bottom



(lint/suspicious/noShorthandPropertyOverrides)